pkgs/swift2objc/lib/src/ast/declarations/compounds/members/property_declaration.dart

```diff
@@ -30,7 +31,14 @@ class PropertyDeclaration implements Declaration, ObjCAnnotatable {
     required this.type,
     this.hasSetter = false,
     this.hasObjCAnnotation = false,
-    this.getterStatements = const [],
-    this.setterStatements = const [],
+    this.getter,
+    this.setter,
   });
 }
+
+class PropertyEncapsulation implements Executable {
```

Comment:
How about `PropertyStatements`

Reply:
yeah I overthought the naming there XD
Done.